data-science/data-engineers.md

```diff
@@ -14,13 +14,13 @@ To figure out which market segments are paying for data, then it may help you to
 
 But even then, it's not enough to just publish useful data on Ocean. **You need to market your data** **assets** to close sales.
 
-Have you tried all these things and are still having trouble making money? Never fear! You can enter one of our [data challenges](https://oceanprotocol.com/challenges) to make sweet OCEAN rewards and build your data science skills.
+Have you tried all these things and are still having trouble making money? Never fear! You can enter one of the [data challenges](https://oceanprotocol.com/challenges) to make sweet OCEAN rewards and build your data science skills.
```

Comment:
`have you tried all these things` , what are they? Publishing in ocean and sorting by sales? Would replace things with steps